FRANKDOC.md

```diff
@@ -10,6 +10,7 @@ The Frank!Doc provides reference information for Frank developers as explained i
 | `@ff.forward` | | Class | Describes a forward (e.g. `success`, `failure`). First argument is name of forward. Second argument is description. |
 | `@ff.tag` | | Class | Tag that classifies the Java class. First argument is tag name, second argument is tag value. |
 | `@ff.default` | `@Default` | Attribute setter | Describes default value. |
+| `@ff.ref` | | Attribute setter | References another Java method from which the description and the default value should be obtained. The reference can give only the class name that has the method, or the class name and the method name separated by a dot. Always use the full class name. |
```

Comment:
Het lijkt mij handiger om van dit ding een annotatie te maken, in plaats van een javadoc tag.
Je kan die annotatie dan een Class-argument geven, die als parameter in simpele classname kan hebben, omdat hij dan mee doet met de imports. Het werkt dan bijvoorbeeld ook beter als er class namen gerefactored worden.

Reply:
Deze PR lost op dat de documentatie niet klopte met de werkelijkheid. Je doet hier een voorstel om de Frank!Doc te verbeteren. Wil je dat ik dat werk in het kader an deze PR ga doen? Ik zou liever aan https://github.com/ibissource/frank-doc/issues/140 gaan werken.